### ISSUE TITLE
add support for time out option

### ISSUE THREAD

**Issue by enoguchi-lmi** [th0r/npm-upgrade#74]
Hello,

Could you please add support for a timeout option such as `ncu --timeout 100000`

**Comment by jkeruzec** [comment]
Yes very usefull especially using npm-upgrade while in remote office with VPN on !

**Comment by ImLunaHey** [comment]
@th0r thank you. ❤
